src/languages/en.ts

```diff
@@ -1869,7 +1869,19 @@ export default {
             errors: {
                 taxRateAlreadyExists: 'This tax name is already in use.',
                 valuePercentageRange: 'Please enter a valid percentage between 0 and 100.',
-                genericFailureMessage: 'An error occurred while updating the tax rate, please try again.',
+                deleteFailureMessage: 'An error occurred while deleting the tax rate, please try again.',
+                updateFailureMessage: 'An error occurred while updating the tax rate, please try again.',
+                createFailureMessage: 'An error occurred while creating the tax rate, please try again.',
+            },
+            deleteTaxConfirmation: 'Are you sure you want to delete this tax?',
+            deleteMultipleTaxConfirmation: ({taxAmount}) => `Are you sure you want to delete ${taxAmount} taxes?`,
+            actions: {
+                delete: 'Delete rate',
+                deleteMultiple: 'Delete rates',
+                disable: 'Disable rate',
+                disableMultiple: 'Disable rates',
+                enable: 'Enable rate',
+                enableMultiple: 'Enable rates',
```

Comment:
@Expensify/marketing could you please review this copy?

Reply:
To confirm, these are just generic error messages where we don't really know exactly what happened? Or is there any more info we have where we can get more specific? Thanks!

Reply:
Yea, these are just generic for when we don't know what happened. Otherwise the server sends specific error messages for the cases we do.

Reply:
I think if needed we can be a bit more specific for `updateFailureMessage`. We can split it:
- toggling the tax rate,
- changing name,
- changing value.

Reply:
I would guess, though, that we'd show the error message right after you take the action that triggered it, so it would be fairly easy to understand what went wrong. Maybe we can stick with the generics here: 

- `deleteFailureMessage`: An error occurred while deleting the tax rate. Please try again or ask Concierge for help.
- `updateFailureMessage`: An error occurred while updating the tax rate. Please try again or ask Concierge for help.
- `createFailureMessage`: An error occurred while creating the tax rate. Please try again or ask Concierge for help.

Reply:
Yea, agreed!

Reply:
I've updated the PR.